react/src/js/react/App.jsx

```diff
@@ -1,23 +1,26 @@
-/**
- * Entry-point for the react code in the app.
- * Allows us to map various URLs to page-components
- */
-
 import React from "react";
 import Reflux from "reflux";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 import PageNotFound from "./help/PageNotFound";
+import PostFeed from "./posts/PostFeed";
+import NewPostButton from "./posts/NewPostButton";
 import Header from "./misc/Header";
 
+
+/**
+ * Entry-point for the react code in the app.
+ * Allows us to map various URLs to page-components
+ */
```

Comment:
I would normally leave file-level comments at the top of the file? dunno

Reply:
Lol I thought it was at the top of classes, like one of the style guides I was looking at said that but now I can't find the link /shrug